salt/modules/mssql.py

```diff
@@ -96,6 +168,8 @@ def tsql_query(query, **kwargs):
     except Exception as err:  # pylint: disable=broad-except
         # Trying to look like the output of cur.fetchall()
         return (("Could not run the query",), (six.text_type(err),))
+    finally:
+        _close_connection(cursor=cur, connect=conn)
```

Comment:
Could you please also update all the functions here to use `_close_connection` instead of `conn.close` and check it's called in a `finally` block?

Reply:
Ok, I'll fix it